src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -0,0 +1,671 @@
+<?php
+/**
+ * Interactivity API: WP_Interactivity_API class.
+ *
+ * @package WordPress
+ * @subpackage Interactivity API
```

Comment:
```suggestion
 * @subpackage Interactivity API
 * @since 6.5.0
```

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/5953/commits/c0f580454a2ddf4db571f3f5e28f5d3a0822a761.